services/network_acl_manager/src/test/java/com/futurewei/alcor/networkaclmanager/config/UnitTestConfig.java

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+package com.futurewei.alcor.networkaclmanager.config;
+
+public class UnitTestConfig {
```

Comment:
Could you please add swagger/SwaggerJsonTest.java to the NACL UT? 

One example is https://github.com/futurewei-cloud/alcor/blob/master/services/security_group_manager/src/test/java/com/futurewei/alcor/securitygroup/swagger/SwaggerJsonTest.java.